dist/ToWords.d.ts

```diff
@@ -5,9 +5,33 @@ export declare class ToWords {
     private options;
     private locale;
     constructor(options?: ToWordsOptions);
+    /**
+     * getting language locale class based on user
+     * passed config options
+     * It contains the mapping for currency, texts and
+     * numberToWord mapping
+     *
+     * @returns {class} - based on selected currency
+     */
     getLocaleClass(): ConstructorOf<LocaleInterface>;
```

Comment:
@apoorvachikara Files inside dist gets deleted on each build. Best is to have code comments in the source files.

Reply:
I put them in the ts file only, the action might create a new dist with comments on it.